website/playground/buttons.js

```diff
@@ -41,16 +37,16 @@ export class ClipboardButton extends React.Component {
   }
 
   render() {
+    const { children } = this.props;
```

Comment:
```suggestion
    const { children, copy, ...rest } = this.props;
```

Reply:
This should allow for removing the three `rest` lines below, right?

Reply:
@lydell, yep, but it falls with:
```bash
[1] SyntaxError: Unexpected token (40:31)
[1]
[1]   38 |
[1]   39 |   render() {
[1] > 40 |     const { children, copy: _, ...rest } = this.props;
```

there is no [`proposal-object-rest-spread`](https://babeljs.io/docs/en/babel-plugin-proposal-object-rest-spread) plugin